scripts/lib/mkcloud-libvirt.sh

```diff
@@ -57,36 +57,85 @@ function libvirt_net_start()
         ip link set mtu 9000 dev $dev
     done
 
+    boot_mkcloud=/etc/init.d/boot.mkcloud
+    boot_mkcloud_d="$boot_mkcloud.d"
+    boot_mkcloud_d_cloud="$boot_mkcloud_d/$cloud"
+
     if [ -z "$NOSETUPPORTFORWARDING" ] ; then
+        # FIXME: hardcoded assumptions about admin net host range
         nodehostips=$(seq -s ' ' 81 $((80 + $nodenumber)))
```

Comment:
Another issue is that this code doesn't deal with forwarding ports to virtual IPs.  But that's still not a reason to -1 this PR, because it's clearly improving things.